source/rpt/parameters_rpt.cc

```diff
@@ -301,3 +301,142 @@ Parameters::RPTReconstructionParameters::parse_parameters(ParameterHandler &prm)
   }
   prm.leave_subsection();
 }
+
+
+void
+Parameters::RPTFEMReconstructionParameters::declare_parameters(
+  ParameterHandler &prm)
+{
+  prm.enter_subsection("fem reconstruction");
+  {
+    prm.declare_entry("mesh type",
+                      "dealii",
+                      Patterns::Selection("dealii|gmsh"),
+                      "Type of mesh used");
+
+    prm.declare_entry("mesh filename",
+                      "reactor.msh",
+                      Patterns::FileName(),
+                      "Imported mesh filename");
+
+    prm.declare_entry("z subdivisions",
+                      "2",
+                      Patterns::Integer(),
+                      "Number of subdivisions of the initial grid in z");
+
+    prm.declare_entry("mesh refinement",
+                      "2",
```

Comment:
Why not 0 by default?

Reply:
When testing different mesh combination, I saw that it wasn't worth it to be under 2; the results of the reconstruction are not that good. Should I leave it that way, or change it?